nailed-it/New Group/PolishLibraryViewController.swift

```diff
@@ -42,21 +45,37 @@ class PolishLibraryViewController: UIViewController, UICollectionViewDelegate, U
 
     @IBAction func onFilter(_ sender: Any) {
         let picker = CZPickerView(headerTitle: "Brands", cancelButtonTitle: "Cancel", confirmButtonTitle: "Confirm")
-        let greenColor = UIColor(red:0.80, green:1.00, blue:0.90, alpha:1.0)
-        let pinkColor = UIColor(red:1.00, green:0.80, blue:0.90, alpha:1.0)
-        picker?.delegate = self
-        picker?.dataSource = self
-        picker?.needFooterView = false
-        picker?.allowMultipleSelection = true
-        picker?.headerBackgroundColor = greenColor
-        picker?.confirmButtonBackgroundColor = greenColor
-        picker?.headerTitleColor = pinkColor
-        picker?.confirmButtonNormalColor = pinkColor
-        picker?.cancelButtonNormalColor = pinkColor
-        picker?.checkmarkColor = pinkColor
-        picker?.show()
+        setUpPicker(picker: picker!, type: "filter")
     }
-
+    
+    @IBAction func onSort(_ sender: Any) {
+        let picker = CZPickerView(headerTitle: "Sort By", cancelButtonTitle: "Cancel", confirmButtonTitle: "Confirm")
+        setUpPicker(picker: picker!, type: "sort")
+    }
+    
+    func setUpPicker(picker: CZPickerView, type: String) {
+        let greenColor = UIColor(red:0.59, green:0.89, blue:0.70, alpha:1.0)
+        let pinkColor = UIColor(red:0.98, green:0.66, blue:0.65, alpha:1.0)
+        picker.delegate = self
+        picker.dataSource = self
+        picker.needFooterView = false
+        if type == "sort" {
+            picker.allowMultipleSelection = false
+            self.sortFilter = "sort"
+        } else {
+            picker.allowMultipleSelection = true
+            self.sortFilter = "filter"
+        }
+        picker.headerBackgroundColor = greenColor
+        picker.confirmButtonBackgroundColor = greenColor
+        picker.headerTitleColor = pinkColor
```

Comment:
nitpick, but can we make the font colors white? like the title font and the button fonts. its just easier to see on a colored background